.github/workflows/ci.yml

```
@@ -35,7 +42,24 @@ jobs:
+          cargo install cargo-set-version
       - name: Run Tests
         run: |
           task test
+      - name: Increase Version
+        id: version_step
+        uses: ./
+        with:
+          github_token: ${{ secrets.GH_TOKEN }}
+          target_branch: ${{ github.ref }}
+          # no_pr: true
+          refresh_minor: true
```

actionlint (1.7.4)

input "refresh_minor" is not defined in action "version-update" defined at "./". available inputs are "github_token", "no_pr", "target_branch", "version_message", "workspaces"

(action)